api/desecapi/tests/base.py

```diff
@@ -158,6 +160,7 @@ def __init__(self, test_case, expected_requests, single_expectation_single_reque
         self.single_expectation_single_request = single_expectation_single_request
         self.expect_order = expect_order
         self.old_httpretty_entries = None
+        self.exit_hook = exit_hook or (lambda: None)
```

Comment:
I think `self.exit_hook = exit_hook` and `if callable(self.exit_hook): self.exit_hook()` down below is cleaner.

Reply:
done